### PR TITLE
Remove obsolete versionadded in 3.4

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Is it possible that this PR has some wrong commits? It looks a bit strange now. Thanks.

**Comment by OskarStark** [comment]
Will check later 👍🏻

File: contributing/community/review-comments.rst [hunk 1]
```diff
@@ -91,34 +91,34 @@ Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrib
 **Don't:** *"I don't like how you wrote this code"* - there is no clear explanation why you
 don't like how it's written.
 
-**Better:** *"I find it hard to read this code as there many nested if statements, can you make it more
-readable? By encapsulating some of it's details or maybe adding some comments to explain the overall logic."* -
+**Better:** *"I find it hard to read this code as there is many nested if statements, can you make it more
```
Comment (HeahDude) [inline contributing/community/review-comments.rst]: there are

File: contributing/community/review-comments.rst [hunk 2]
```diff
@@ -91,34 +91,34 @@ Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrib
...
+readable? By encapsulating some of its details or maybe adding some comments to explain the overall logic."* -
```
Comment (HeahDude) [inline contributing/community/review-comments.rst]: the?

File: contributing/community/review-comments.rst [hunk 3]
```diff
@@ -91,34 +91,34 @@ Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrib
...
 You explain why you find the code hard to read *and* give some suggestions for improvement.
 
 If a piece of code is in fact wrong, explain why:
 
-* ``This code doesn't comply with Symfony's CS rules. Please see [...] for details``.
+* "This code doesn't comply with Symfony's CS rules. Please see [...] for details."
 
-* ``Symfony 3 still uses PHP 5 and doesn't allow the usage scalar type-hints.``.
+* "Symfony 3 still uses PHP 5 and doesn't allow the usage scalar type-hints."
```
Comment (HeahDude) [inline contributing/community/review-comments.rst]: the usage of

File: contributing/community/review-comments.rst [hunk 4]
```diff
@@ -91,34 +91,34 @@ Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrib
...
-    * That change would introduce too many merge conflicts when merging up Symfony branches.
-      In the past we've always rejected changes like this.
+* "That change would introduce too many merge conflicts when merging up Symfony branches.
+  In the past we've always rejected changes like this."
 
-    * I profiled this change and it hurts performance significantly (if you don't profile, it's an opinion, so we can ignore)
+* "I profiled this change and it hurts performance significantly" - if you don't profile, it's an opinion, so we can ignore
 
-    * Code doesn't match Symfony's CS rules (e.g. use ``[]`` instead of ``array()``)
+* "Code doesn't match Symfony's CS rules (e.g. use ``[]`` instead of ``array()``)"
```
Comment (HeahDude) [inline contributing/community/review-comments.rst]: Can be removed now I guess

**Review by HeahDude — COMMENTED** [review]
Apart minor comments, big :+1: from me, thanks!

**Comment by HeahDude** [comment]
Also we should target 3.4 here.

**Comment by HeahDude** [comment]
Then remove useless notices in another PR targetting 4.2.

**Comment by OskarStark** [comment]
> Is it possible that this PR has some wrong commits? It looks a bit strange now. Thanks.

Hmm you are right, lets close it for now, I will check that

**Pull request closed without merging**